### ISSUE TITLE
Feature Request: display video duration on video page

### ISSUE THREAD

**Issue by yasu4** [TeamNewPipe/NewPipe#1169]
- [.] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [.] I checked if the issue/feature exists in the latest version.
When opening a link to the video, if possible, i'd like to know video duration before starting watching it.

**Comment by mauriciocolli** [comment]
Duplicate of #609.
